### ISSUE TITLE
Completion doesn't show type of variable identifiers

### ISSUE THREAD

**Issue by fatih** [zchee/deoplete-go#17]
This is the output when I use deoplete-go:

![image](https://cloud.githubusercontent.com/assets/438920/12869971/735c658e-cd37-11e5-9876-c8a2defecc5a.png)

And here is the info from gocode when I call it with `c-x c-o`. As you see it shows that the type is:

![image](https://cloud.githubusercontent.com/assets/438920/12869973/82761808-cd37-11e5-8a2c-03532337fee1.png)

As you see the type information is not available for deoplete.

**Comment by fatih** [comment]
I think this clause should be fixed and you should provide more information: 

https://github.com/zchee/deoplete-go/blob/master/rplugin/python3/deoplete/sources/go.py#L90

**Comment by zchee** [comment]
@fatih Thanks! 
I want to debug for this problem
Could you post the this `.go` file?

**Comment by nhooyr** [comment]
@zchee what is there to debug. Fatih just wants us to format the information like

``` python
abbr = _class + ' ' + complete['name'] + ' ' + complete['type']
```

**Comment by nhooyr** [comment]
closed by #23

**Comment by zchee** [comment]
Solved https://github.com/zchee/deoplete-go/pull/23
Close.
